jetty9-compat-base/src/test/java/com/google/apphosting/vmruntime/jetty9/JettyRunner.java

```diff
@@ -166,9 +167,8 @@ public void doStart() throws Exception {
       httpConfig.setSendServerVersion(true);
       httpConfig.setSendDateHeader(false);
       httpConfig.setDelayDispatchUntilContent(false);
-      GoogleRequestCustomizer requestCustomizer = new com.google.apphosting.jetty9.GoogleRequestCustomizer();
-      requestCustomizer.setHttpPort(port);
-      requestCustomizer.setHttpsPort(Integer.parseInt("443"));
+      GoogleRequestCustomizer requestCustomizer =
+          new GoogleRequestCustomizer(port, Integer.parseInt("443"));
```

Comment:
I still don't understand why you need the `Integer.parseInt("443")` conversion rather than just passing
the literal `443`...